packages/react/src/River/RiverBreakout.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {Ref, forwardRef} from 'react'
+import clsx from 'clsx'
+import {RiverProps} from '.'
+import {useAnimation} from '../animation'
+import styles from './River.module.css'
+import {Heading} from '../Heading'
```

Comment:
nit: suggestion on ordering

```suggestion
import {Heading} from '../Heading'

import styles from './River.module.css'
```